rs/nns/governance/src/proposals/manage_neuron.rs

```diff
@@ -0,0 +1,437 @@
+use crate::{
+    pb::v1::{
+        Account, ManageNeuron, SelfDescribingValue, Subaccount, Topic, Visibility,
+        manage_neuron::{
+            AddHotKey, ChangeAutoStakeMaturity, ClaimOrRefresh, Command, Configure, Disburse,
+            DisburseMaturity, DisburseToNeuron, Follow, IncreaseDissolveDelay, JoinCommunityFund,
+            LeaveCommunityFund, Merge, MergeMaturity, NeuronIdOrSubaccount, RefreshVotingPower,
+            RegisterVote, RemoveHotKey, SetDissolveTimestamp, SetFollowing, SetVisibility, Spawn,
+            Split, StakeMaturity, StartDissolving, StopDissolving, claim_or_refresh, configure,
+            disburse, set_following,
+        },
+    },
+    proposals::self_describing::{LocallyDescribableProposalAction, ValueBuilder},
+};
+
+use ic_nns_common::pb::v1::NeuronId;
+
+fn account_identifier_to_hex(acc: &icp_ledger::protobuf::AccountIdentifier) -> String {
```

Comment:
This seems to duplicate [existing library code][hex]? Can you use the existing method instead? If not, don't worry about it.

[hex]: https://sourcegraph.com/github.com/dfinity/ic@8146e4d90c20e0999c1b9826bcb905e929c190e7/-/blob/rs/ledger_suite/icp/src/account_identifier.rs?L129